roles/gaia/tasks/main.yml

```diff
@@ -66,7 +66,18 @@
   # TODO: should we overwrite?
   shell: |
     PATH=$PATH:/usr/local/go/bin:{{ ansible_env.HOME }}/go/bin
-    gaiad init {{inventory_hostname}} --home {{gaiad_home}}
+    gaiad init {{inventory_hostname}} --home {{gaiad_home}} --chain-id "theta-devnet"
+
+- name: create validator
+  when: gaiad_create_validator
+  shell: |
+    PATH=$PATH:/usr/local/go/bin:{{ ansible_env.HOME }}/go/bin
+    gaiad keys add validator --keyring-backend="test" --home {{gaiad_home}}
+    gaiad add-genesis-account validator '10000000000stake,1000000000000uatom' --home {{gaiad_home}} --keyring-backend="test"
+    gaiad add-genesis-account cosmos1hxv7mpztvln45eghez6evw2ypcw4vjmsmr8cdx 1000stake,1000000000uatom  --home {{gaiad_home}}
+    gaiad add-genesis-account cosmos1cc5mtuje84sx5dsqw4gtr4w8lthpzw02rjjnew 1000stake,1000000000uatom --home {{gaiad_home}}
+    gaiad gentx validator 8000000000stake --keyring-backend="test" --home {{gaiad_home}} --chain-id "theta-devnet"
```

Comment:
Should all this all be in variables?

Reply:
Yes these should all be variables. Is it possible for you to move them to vars?

Reply:
I have turned them into variables and defined in inventory file please move them if they don't belong there.